Lesson6 DZ/src/Land.java

```diff
@@ -1,4 +1,12 @@
 public class Land extends Transport {
+    public int getWheels() {
```

Comment:
pls refactor it. First there should be fields declaration (int wheels; ...), then getters